db/migrate/20190213200342_add_admin_column_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminColumnToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :is_admin, :boolean
```

Comment:
Convention for ruby is for boolean properties to not start with "is", instead here you would just say `admin`, and then in your model Rails will actually expose it for you so that you can say `user.admin?`

Reply:
I went in and made a migration with change_table to rename the column